src/test/java/uk/ac/cam/cl/dtg/isaac/api/AuthenticationFacadeIT.java

```diff
@@ -0,0 +1,42 @@
+package uk.ac.cam.cl.dtg.isaac.api;
+
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.servlet.http.HttpSession;
+import org.eclipse.jetty.http.CookieCompliance;
+import org.eclipse.jetty.server.*;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import uk.ac.cam.cl.dtg.segue.api.AuthenticationFacade;
+
+import static org.easymock.EasyMock.*;
+import static org.eclipse.jetty.http.HttpCookie.SAME_SITE_LAX_COMMENT;
+import static org.junit.Assert.assertEquals;
+
+@RunWith(PowerMockRunner.class)
```

Comment:
What is PowerMock being used for in this test?

Reply:
Ah, that was from my incomplete attempts at a functional Response object for the logout cookie. Now that those have been cleaned out, this isn't needed for the small remaining test.